ref/perf-benchmarks.md

```diff
@@ -1,21 +1,21 @@
 The following benchmarks give an overview of the performance that you can achieve with Anbox Cloud.
 
-The benchmarks were performed using the `amc benchmark` utility as described in [How to run benchmarks](https://discourse.ubuntu.com/t/benchmarking-a-deployment/17770). The results describe the maximum number of parallel running containers (column "# Containers") delivering a stable frame rate (column "Avg. FPS"). Running more containers either gives too high variation in the provided frame rate or is not possible due to other hardware limitations (system memory, GPU memory, ...).
+The benchmarks were performed using the `amc benchmark` utility as described in [How to run benchmarks](https://discourse.ubuntu.com/t/benchmarking-a-deployment/17770). The results describe the maximum number of parallel running containers (column **# Containers**) delivering a stable frame rate (column **Avg. FPS**). Running more containers either gives a very high variation in the provided frame rate or is not possible due to other hardware limitations such as system memory, GPU memory etc.
 
-All benchmarks include rendering and video encoding. On machines/VMs without a GPU, rendering and video encoding are performed in software on the CPU.
+For most of the benchmarks below, a special version of the [BombSquad](https://www.froemling.net/apps/bombsquad) application was used. This version runs in a demo mode in which the game provides random and automated simulated gameplay. This allowed the benchmark to simulate a real-world scenario where actual users would play the game, instead of deriving from a static game scene without much variation across the Android instances.
```

Comment:
```suggestion
For most of the benchmarks below, a special version of the [BombSquad](https://www.froemling.net/apps/bombsquad) application was used. This version runs in a demo mode in which the game provides random and automated simulated gameplay. This allowed the benchmark to simulate a real-world scenario where actual users would play the game, instead of deriving from a static game scene without much variation.
```
I feel like 'the Android instances' at the end of the sentence is a bit confusing. What do you think?

Reply:
I dont' find it confusing but in any case, it does not add any value to the statement. I agree to removing it.